addons/web/controllers/dataset.py

```diff
@@ -56,9 +56,6 @@ def resequence(self, model, ids, field='sequence', offset=0, context=None):
         if context:
             request.update_context(**context)
         m = request.env[model]
-        if not m.fields_get([field]):
-            return False
-        # python 2.6 has no start parameter
-        for i, record in enumerate(m.browse(ids)):
-            record.write({field: i + offset})
-        return True
```

Comment:
Why not deprecated this one ? Is it still used ?

Reply:
Yep, still used by website